src/components/media-player/ha-media-player-browse.ts

```diff
@@ -246,6 +248,16 @@ export class HaMediaPlayerBrowse extends LitElement {
             ],
             replace: true,
           });
+        } else if (
+          err.code === "entity_not_found" &&
+          UNAVAILABLE_STATES.includes(this.hass.states[this.entityId]?.state)
```

Comment:
Do we need this check still?

Reply:
Yea. If the entity isnt found for another reason then we still just show the error. Not sure if there is another reason but guessing it could be possible to ask for an entity that doesn't exist at all